2025/build-tooling/demo/final/src/vite-env.d.ts

```diff
@@ -1,3 +1,4 @@
 /// <reference types="vite/client" />
+/// <reference types="@vitest/browser/matchers" />
```

Comment:
Double check whether this is needed instead

https://devtopia.esri.com/WebGIS/arcgis-web-components/blob/55d8cd28886b8307310ddd59c8b59e970b8e0270/packages/map-packages/map-components/src/lumina.ts#L13C1-L14C63

Reply:
Current approach works and follows the doc: https://vitest.dev/guide/browser/#typescript